universe/Condition.cpp

```diff
@@ -3279,8 +3280,14 @@ std::string Condition::PlanetEnvironment::Description(bool negated/* = false*/)
         }
     }
     std::string species_str;
-    if (m_species_name)
-        species_str = m_species_name->Description();
+    if (m_species_name) {
+        if (ValueRef::ConstantExpr(m_species_name)) {
+            species_str = m_species_name->Eval();
```

Comment:
Actually nevermind about ->Description(); I see ->Eval() used in various places already.